.github/workflows/tests.yaml

```diff
@@ -49,7 +64,7 @@ jobs:
       - name: Check license headers
         run: |
           set -e
-          addlicense -l apache -c 'The Rekor Authors' -v *
+          addlicense -l apache -c 'The Sigstore Authors' -v -check *
```

Comment:
I think the next line is useless if you add `-check` here.
The reason why I did this without `-check` is so that we in-place update the files, and the next command `git diff --exit-code` causes step to fail because there's a diff.

But if you fail here w/ -check, the next command won't execute because of the `set -e` above?

Reply:
@ahmetb you are correct! thanks for the catch
I did a test in the GH actions and even if we set or not the flag `set -e` if a command return error it will abort the execution

I will update both repos i did the change to use the original way.